jgiven-tests/src/test/java/com/tngtech/jgiven/junit5/JUnit5ExecutorTest.java

```diff
@@ -0,0 +1,72 @@
+package com.tngtech.jgiven.junit5;
+
+import com.tngtech.jgiven.GivenScenarioTest;
+import com.tngtech.jgiven.JGivenScenarioTest;
+import com.tngtech.jgiven.tags.FeatureJUnit5;
+import com.tngtech.jgiven.tags.Issue;
+import com.tngtech.jgiven.testframework.ThenTestFramework;
+import com.tngtech.jgiven.testframework.WhenTestFramework;
+import org.junit.Test;
+
+@FeatureJUnit5
+public class JUnit5ExecutorTest
+    extends JGivenScenarioTest<GivenScenarioTest<?>, WhenTestFramework<?>, ThenTestFramework<?>> {
+
+
+    @Test
+    public void tests_with_scenario_modifications_in_after_method() {
+        given().junit5_tests_with_scenario_modifications_in_after_method();
+        when().the_test_class_is_executed_with_JUnit5();
+        then().each_scenario_contains_$_cases(2);
+    }
+
+    @Test
+    @Issue("#25")
+    public void test_classes_with_only_ignored_test_result_in_a_valid_report() {
```

Comment:
Is it somehow possible to give this method some better name? Needed some time to figure out which word groups belong to the same context. Like "test_classes_which_consist_of_only_ignored_tests" or something.